turbopack/crates/turbopack-ecmascript/src/analyzer/imports.rs

```diff
@@ -402,7 +437,7 @@ impl Visit for Analyzer<'_> {
         let i = self.ensure_reference(
             export.span,
             export.src.value.clone(),
-            symbol.unwrap_or(ImportedSymbol::Exports),
+            symbol.unwrap_or(ImportedSymbol::ReexportAll),
```

Comment:
That looks wrong to me. When I `export * from "module"` I want to reference the `Exports` part of that because it includes all exports. Why do we need `ReexportAll` at all?